src/routes/index.ts

```diff
@@ -227,6 +235,15 @@ router.route(config.BENEFICIAL_OWNER_OTHER_URL + config.ID)
   .post(...validator.beneficialOwnerOther, checkValidations, beneficialOwnerOther.update);
 router.get(config.BENEFICIAL_OWNER_OTHER_URL + config.REMOVE + config.ID, authentication, navigation.hasBeneficialOwnersStatement, beneficialOwnerOther.remove);
 
+router.route(config.BENEFICIAL_OWNER_OTHER_WITH_PARAMS_URL + config.ID)
+  .all(
+    authentication,
+    navigation.hasBeneficialOwnersStatement
+  )
+  .get(beneficialOwnerOther.getById)
+  .post(...validator.beneficialOwnerOther, checkValidations, beneficialOwnerOther.update);
+router.get(config.BENEFICIAL_OWNER_OTHER_WITH_PARAMS_URL + config.REMOVE + config.ID, authentication, navigation.hasBeneficialOwnersStatement, beneficialOwnerOther.remove);
```

Comment:
can you group this remove route with the non-params one and separate from other groups with blank line. eg, line 244-245 needs blank line separator so we can see the routes more easily

Reply:
Updated this to be grouped.